core/platform/email/mailgun_email_services_test.py

```diff
@@ -53,3 +58,129 @@ def test_sending_email(self):
             mailgun_email_services.send_mail(
                 feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
                 'subject', 'body', 'html', bcc_admin=False)
+
+
+    def test_sending_email_data(self):
```

Comment:
Can this name be more detailed... Basically the name should convey what the test is doing, any specific conditions, and the expected outcome.

Reply:
Ditto, for all test function names.